controllers/controller.js

```diff
@@ -0,0 +1,12 @@
+const {topicsMod} = require("../models/model")
+const apiFile = require("../endpoints.json")
+exports.topicsCont = (req,res,next)=>{    
```

Comment:
naming convention for your models and controllers would normally be something like getTopics and fetchTopics